frontend/src/app/address/address.page.html

```diff
@@ -23,9 +23,12 @@
         </ion-col>
         <ion-col size="4" class="ion-text-end">
           <ngx-qrcode
-            [(qrc-value)]="qrcode"
-            qrc-element-type="url"
-          ></ngx-qrcode>
+            elementType="url"
+            [value]="qrcode"
+            cssClass="aclass"
```

Comment:
"aclass" is not defined in css.

Reply:
Thanks, I just pated example in the library document... I will remove it.